beacon-chain/p2p/testing/p2p.go

```diff
@@ -287,12 +288,14 @@ func (p *TestP2P) Send(ctx context.Context, msg interface{}, topic string, pid p
 
 	if topic != "/eth2/beacon_chain/req/metadata/1" {
 		if _, err := p.Encoding().EncodeWithMaxLength(stream, msg); err != nil {
+			stream.Reset()
```

Comment:
Please handle this error from stream.Reset(). (Static analysis will not allow the build to complete with unhandled errors)